docs/stencil-docs/storefront-customization/theme-accessibility.md

```diff
@@ -0,0 +1,178 @@
+2. Launch Page Builder and select the image widget.
+3. Enter the alt text.
+4. Click **Save**.
+
+![Image Widget](https://raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/images/accessibility_image_widget.png "Image Widget")
+
+### Aria-label attribute
+There are other options for providing text for images. Developers can use the aria-label attribute, a [Web Content Accessibility Guidelines (WCAG)](https://www.w3.org/WAI/standards-guidelines/wcag/) technique that provides assistive technology with the aria-label text instead of the image alt text. 
+
+The aria-label attribute uses an ID reference value that matches the ID element attribute to associate an element with text. Screen readers use the text of one or more identified elements that can be referenced elsewhere on the page.
+
+**Aria-label example**
+
+The example below displays how to use an aria-label for two elements considered to be one image.
+
+```html
+<div role="img" aria-label="Cart images">
+
+<img src = "giftCard.png" alt=""/>
+<img src = "Checkout.png" alt=""/>
+</div>
+```
+
+## Text accessibility
+Text accessibility involves making content readable and understandable. You can achieve text accessibility using the following methods:
+* Color contrast
+* Headings
+* Font size and text alignment
+* Text links
+
+### Color contrast
+Color contrast is an essential factor in visual accessibility.
+
+<div class="HubBlock--callout">
+<div class="CalloutBlock--info">
+<div class="HubBlock-content">
+
+> ### Note
+> According to [WCAG 2.1](https://www.w3.org/TR/WCAG21/), your ecommerce website must have minimum color contrast ratios of 4.5.1 for standard text and 3:1 for large text. To ensure you meet WCAG 2.1 guidelines, we recommend presetting these guidelines, using [Lighthouse](https://developers.google.com/web/tools/lighthouse) or Chrome's [Accessibility Developer Tools](https://chrome.google.com/webstore/detail/accessibility-developer-t/fpkknkljclfencbdbgkenhalefipecmb/reviews?hl=en).
+
+</div>
+</div>
+</div>
+
+Use contrasting colors for better readability and clarity. Pure black text on a white background can cause eye strain. Instead, try black text on light blue, pale green, or gray background. The following example demonstrates good color contrasting and has a 14.9.1 contrast ratio.
+
+**Good contrast ratio**
+![Good contrast](https://raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/images/accessibility_good_color_contrast.png "Black text on a light gray background")
+
+It is essential to avoid colors that are too close together. For example, avoid green and black, green and gray, and green and red. Alternatively, it is good practice to use light text on dark backgrounds or dark text on light backgrounds. The example below shows poor color contrasting because the black text is on a green background. The color constrast ratio is 3.8.1. It would have been better to use white text on a dark green background.
```

Comment:
I'm unclear what you mean by colors that are "too close together." Add more precise language on the meaning of close. Close in what way?